### ISSUE TITLE
[Bug] SQLite > Maria migration error (2.x->3.x)

### ISSUE THREAD

**Issue by krohrsb** [zurdi15/romm#688]
**RomM version**
3.0.0

**Describe the bug**
SQL Migration script encounters an error.

```
.....(prior logs).....
INFO:     [RomM][alembic.runtime.migration] Running upgrade 1.8.2 -> 1.8.3, update to 1.8.3
INFO:     [RomM][alembic.runtime.migration] Running upgrade 1.8.3 -> 2.0.0, update to 2.0.0
INFO:     [RomM][alembic.runtime.migration] Running upgrade 2.0.0 -> 0009_models_refactor, empty message
INFO:     [RomM][alembic.runtime.migration] Running upgrade 0009_models_refactor -> 0010_igdb_id_integerr, empty message
INFO:     [RomM][alembic.runtime.migration] Running upgrade 0010_igdb_id_integerr -> 0011_drop_has_cover, empty message
INFO:     [RomM][alembic.runtime.migration] Running upgrade 0011_drop_has_cover -> 0012_add_regions_languages, empty message
INFO:     [RomM][alembic.runtime.migration] Running upgrade 0012_add_regions_languages -> 0013_upgrade_file_extension, empty message
INFO:     [RomM][alembic.runtime.migration] Running upgrade 0013_upgrade_file_extension -> 0014_asset_files, empty message
Traceback (most recent call last):
....(stacktrace)....
mariadb.ProgrammingError: statement (1) doesn't match the number of data elements (0).
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
sqlalchemy.exc.ProgrammingError: (mariadb.ProgrammingError) statement (1) doesn't match the number of data elements (0).
[SQL: INSERT INTO roms (id, igdb_id, p_igdb_id, sgdb_id, p_sgdb_id, platform_slug, p_name, file_name, file_name_no_tags, file_extension, file_path, file_size, file_size_units, name, slug, summary, path_cover_s, path_cover_l, revision, tags, multi, files, url_cover, url_screenshots, path_screenshots, regions, languages) VALUES (1065, 40779, "", null, "", "atari2600", "", "Atlantis 2.BIN", "Atlantis 2", "BIN", "roms/atari2600", 4.0, "KB", "Atlantis II", "atlantis-ii", "Many game companies in the 1980\'s held contests in an attempt to boost sales of their games and give their fans something fun to compete for. For example, there is Activision\'s Enduro: Race for Riches, Data Age\'s Bermuda Triangle Replica Artifact, and Parker Brothers\' Super Cobra Flight Jacket. Not surprisingly, Imagic also held several contests, one of which is \"Defend Atlantis\".
Unfortunately for Imagic, there were a lot of good Atlantis players. In fact, more than four individuals maxed out the score, so Imagic had to figure out a way to decide who the top four would be. They created a special contest version of Atlantis, that they called Atlantis II, that these top scorers would use for a tie-breaker.
Atlantis II is basically the same as Atlantis, except that it\'s faster, harder, and lower scoring. Contestants were sent a letter telling them that they qualified for this shoot off. They were to recieve in the mail the Atlantis II cartridge, along with the snorkeling set and t-shirt.
According to Imagic, \"We\'ve designed this contest version of Atlantis just for you experts. Those tricky Gorgons will still thwart you at every turn, but now the battle will be more fierce and the points scored for direct hits will be less.
Are you expert enough to meet this challenge and save Atlantis from the dreaded Gorgon death ray? If you\'re brave enough, read on.\" Contestants were to play the game for only two days and then send in their high scores.", "atari2600/Atlantis%20II/cover/small.png?timestamp=1704605184.054055", "atari2600/Atlantis%20II/cover/big.png?timestamp=1704605184.367968", "", '[]', 0, '[]', "https://images.igdb.com/igdb/image/upload/t_thumb/co5vji.jpg", '[]', '[]', '[]', '[]')]
(Background on this error at: https://sqlalche.me/e/20/f405)
ERROR:    [init][2024-03-12 09:30:26] Something went horribly wrong with our database
```

Perhaps simply a data escaping issue.

**To Reproduce**
1. Follow migration guide from 2.x to 3.x
2. Start it all up
3. (Maybe specifically have this one game in the library)

**Expected behavior**
DB Migrated successfully

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by gantoine** [comment]
Ok yeah I see the issue, will have it fixed in the next release.

**Comment by gantoine** [comment]
@krohrsb can you try with the latest 3.0.1 release?

**Comment by krohrsb** [comment]
Confirmed this is fixed via #702
